kie-maven-plugin/src/main/java/org/kie/maven/plugin/BuildMojo.java

```diff
@@ -180,20 +175,30 @@ private void buildDrl() throws MojoFailureException {
             ResultsImpl messages = (ResultsImpl)kieBuilder.getResults();
 
             List<Message> errors = messages != null ? messages.filterMessages( Message.Level.ERROR): Collections.emptyList();
+
+            if (container != null) {
+                Optional<Map<String, Object>> optionalKieMap = getKieMap();
```

Comment:
Don't use Optional for Collections, reference: https://stackoverflow.com/a/26328555

Reply:
Fixed